src/components/Apps/UserConfirmDialog/index.vue

```diff
@@ -98,7 +98,6 @@
             type="primary"
             @click="handleFaceCapture"
           >
-            开始人脸识别
             {{ this.$tc('VerifyFace') }}
           </el-button>
         </el-col>
```

Comment:
There do not appear to be any significant differences or irregularities between the two versions of this code snippet from 2021-09-01. This seems like an existing piece of software code that has been translated into Chinese and slightly adapted due to localization needs.

However, if there's more detail required about why the changes were made or what aspects could benefit from being further optimized for Chinese user interface design, please provide specific details so I can refine my response accordingly.